### DISCUSSION TITLE
[Bug]: Tools, Actions, Retrieval and Code Interpreter are no longer visible in OpenAI Assistants

### DISCUSSION THREAD

**Discussion by virtuman** [danny-avila/LibreChat#2101] [Troubleshooting]
### What happened?

after last merge, all capabilities are no longer available for Tools, Actions, Retrieval and Code Interpreter

### Steps to Reproduce

use openai, 
select create new assistant, or edit existing, 
-> none of the capabilities are available at the end of the form

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Try rebuilding, how do you update?

**Reply by danny-avila** [reply]
nvm I just reproduced it, going to open the prev issue.

**Reply by danny-avila** [reply]
Thanks for reporting this, it's merged to main now. maybe 15 min till docker images are rebuilt
